api/envoy/api/v2/core/address.proto

```diff
@@ -20,6 +20,9 @@ message Pipe {
   // Paths starting with '@' will result in an error in environments other than
   // Linux.
   string path = 1 [(validate.rules).string = {min_bytes: 1}];
+
+  // [#not-implemented-hide:]
```

Comment:
Why is this `[#not-implemented-hide:]`, it seems to be implemented below?

Reply:
Updated